.github/workflows/soak.yml

```diff
@@ -1,31 +1,40 @@
-name: E2E Soak Test
+name: on_demand_soak_test
 on:
-  schedule:
-    - cron: '0 0 * * *'
   workflow_dispatch:
     inputs:
-      cl_branch_ref:
-        description: Chainlink repo branch to integrate with
+      base64_config:
+        description: Your .toml file as base64
         required: true
-        default: develop
-        type: string
-      ttl:
-        description: The total time for the test to live
+      cl_image_tag:
+        description: Core image tag
         required: true
-        default: 3h
+        default: develop
         type: string
-      node_count:
-        description: The number of chainlink nodes to use
+      test_runner_tag:
+        description: Remote runner tag that will run the tests
+        default: develop
         required: true
-        default: '5'
         type: string
-
 env:
-  REF_NAME: ${{ github.head_ref || github.ref_name }}
-  ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-solana-tests:${{ github.sha }}
+  CL_ECR: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
   CONTRACT_ARTIFACTS_PATH: contracts/target/deploy
+  MOD_CACHE_VERSION: 1
 
 jobs:
+  check_test_compilation:
+    name: Check integration test compilation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+      - name: Setup go
+        uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
+        with:
+          go-version-file: "go.mod"
+          check-latest: true
+          cache: true
+      - run: cd ./integration-tests && go test -run=xxx ./...
```

Comment:
probably not needed on a manual soak? it might be safe to assume that the test compiles 🤷🏼